server/lib/configuration.js

```diff
@@ -438,6 +438,32 @@ const conf = module.exports = convict({
       'tiny'
     ]
   },
+  scopedKeys: {
+    enabled: {
+      default: false,
+      doc: 'Enable Scoped Key OAuth features',
+      env: 'SCOPED_KEYS_ENABLED',
+      format: Boolean,
+    },
+    validation: {
+      default: {
+        'https://identity.mozilla.org/apps/lockbox': {
+          redirectUris: [
+            'https://2aa95473a5115d5f3deb36bb6875cf76f05e4c4d.extensions.allizom.org',
+            'lockbox://redirect.ios'
```

Comment:
Is this actually the URL that's proposed for use on iOS?

Reply:
@rfk yeap!

Reply:
I have opinions but I'll raise them out-of-band from this PR :-P

Reply:
I will remove the `'lockbox://redirect.ios'` for now so we can discuss it, we can tweak the config later